code/game/gamemodes/cult/hell_universe.dm

```diff
@@ -60,8 +60,9 @@ In short:
 
 	runedec += 9000	//basically removing the rune cap
 
-	for(var/turf/T in world)
-		T.update_lumcount(1, 255, 0, 0, 0)
+	spawn()
```

Comment:
It's no longer done right at the start of the cascade. Pomf added the line 'set background = 1', which means this proc is done in the background simultaneously with other procs. Meaning that adding in any spawn() or sleep() should be unnecessary.